foundation/foundation-mda/src/main/resources/templates/data-delivery-pyspark/pipeline.pom.xml.vm

```diff
@@ -24,7 +24,7 @@
                 <artifactId>fermenter-mda</artifactId>
                 <configuration>
                     <basePackage>${moduleArtifactIdPythonCase}</basePackage>
-                    <profile>data-delivery-pyspark</profile>
+                    <profile>data-delivery-pyspark-pipeline</profile>
                     <propertyVariables>
```

Comment:
What prompted this change?

Reply:
This is actually core fix to remove duplication. According to profile.json, data-delivery-pyspark profile had unnecessary profile references which led to duplication. Just importing data-delivery-pyspark-pipeline can be sufficient.